apps/wallet-mobile/src/navigationOptions.tsx

```diff
@@ -1,15 +1,13 @@
 import {Platform} from 'react-native'
 
-import {COLORS} from './theme'
-
-export const defaultNavigationOptions = {
-  headerStyle: {
-    backgroundColor: COLORS.BACKGROUND_BLUE,
-    elevation: 0,
-    shadowOpacity: 0,
-  },
-  headerTintColor: '#fff',
-} as const
+// export const defaultNavigationOptions = {
```

Comment:
can it be deleted?

Reply:
yes! I think so but was not sure if is needed 😅 will delete it for good now